community/README.md

```diff
@@ -0,0 +1,18 @@
+![Logo](https://storage.googleapis.com/model_garden_artifacts/TF_Model_Garden.png)
+
+# TensorFlow Community Models
+
+We provide a curated list of the GitHub repositories with machine learning models and implementations powered by TensorFlow 2.
+
+Unlike our "[official](https://github.com/tensorflow/models/tree/master/official)" and "[research](https://github.com/tensorflow/models/tree/master/research)" repositories, we do not maintain the GitHub repositories listed here. Their respective maintainers support them.
+
+## Models / Implementations
+
+| Use case | GitHub Repository | Paper | Description | Maintainer(s) |
+|----------|-------------------|-------|-------------|---------------|
+| Computer Vision | [Mask R-CNN](https://github.com/NVIDIA/DeepLearningExamples/tree/master/TensorFlow2/Segmentation/MaskRCNN) | Mask R-CNN (https://arxiv.org/abs/1703.06870) | A script and recipe to train the Mask R-CNN model | [NVIDIA](https://github.com/NVIDIA) |
+| Computer Vision | [U-Net Medical Image Segmentation](https://github.com/NVIDIA/DeepLearningExamples/tree/master/TensorFlow2/Segmentation/UNet_Medical) | U-Net: Convolutional Networks for Biomedical Image Segmentation (https://arxiv.org/abs/1505.04597) | A script and recipe to train U-Net Medical | [NVIDIA](https://github.com/NVIDIA) |
+
+
+## Contributions
+If you have a suggestion for the community model repository, please submit a pull request.
```

Comment:
Who will be reviewers? How about adding our github ids?

Reply:
The Model Garden team will be reviewers. There is no way for sending a direct/private message using a GitHub username. Other model zoos are also using a pull request to accept a new contribution.

Reply:
OK.